p5/io/local_data.py

```diff
@@ -0,0 +1,44 @@
+class table:
+
+	def __init__(self,PATH):
+		try:
+			file = open(PATH,'r')
+		except:
+			print("No file detected.")
+			exit(1)
+		lines = file.readlines()
+		data = []
+		for line in lines:
+			fragment = line.split(',')
+			x = fragment[len(fragment)-1]
+			x = x[:-1]
+			fragment[len(fragment)-1] = x
+			data.append(fragment)
+		self.data = data
+
+	def get_row_count(self):
+		return (len(self.data))
+
+	def get_column_count(self):
+		return (len(self.data[0]))
+
+	def get_column(self,name):
+		count = 0
+		try:
+			for i in self.data[0]:
+				if i == name:
+					break
+				count = count + 1
+			column = []
+			for item in self.data:
+				column.append(item[count])
+			return column
+		except:
```

Comment:
I think it's better to just throw an error if the designated column is not found instead of printing to stdout, because it's harder for the user to realize & handle the latter. Usually if a column name is not found then there is a bug with user code, so if the user program crashes that's a clearer indicator compared to an output in stdout. If there isn't a bug with user code, using an error also allows the user to handle it themselves.